qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -95,7 +104,7 @@ def join_network(self):
         wait_bitcoinds()
         self.setup_network(False)
 
-    def main(self):
+    def main(self,argsOverride=None,bitcoinConfDict=None,wallets=None):
```

Comment:
these new parameters need some brief introduction, here or in another suitable place (docstring)